### DISCUSSION TITLE
When approximately should we expect version 2 to replace version 1?

### DISCUSSION THREAD

**Discussion by alex4998** [Live-Charts/LiveCharts2#422] [Q&A]
Hi,
   I'm monitoring this project for quite a while (about 4-5 month) and using the beta version of it for WinUI project. I wonder what the plans are for releasing this product to wider audience and essentially replacing the first version?
   Is it possible to use "geared" with LiveChart2 WinUI? If yes, can you give directions - how, if no, what are the plans for that?

Thank you,
Alex

**Comment by alex4998** [comment]
I'm kind of really worry how much alive this project is...

**Comment by beto-rodriguez** [comment]
@alex4998 v2 is a huge step, LiveCharts is now multiplatform, it was a huge step, the amount of effort in v2 is not comparable with v0, the difference is just huge, v2 is a self sufficient framework.

I do not want to rush the project, and I really don't see the need on that, the last release of `v2` was 2 months ago, I do not think that is a `dead` project.

Yes there is going to be a high performance package for v2. actually out of the box v2 is much faster than v0, the bottle neck of v0 was in wpf, now we are not running over wpf, we handle it all, the open source version of v2 is at least 10-15 times faster than v0.

If you ask me for a date, I would say it should be ready 6 months ago...

Everything takes its time, I am testing the new high performance package and building more features. I just do not want to rush `v2` I want a high quality product, and that takes some time to build.

I hope you understand.

**Comment by Albert634515** [comment]
I don't think the project is dead either and super nice.
Extra features are always nice, but it would be nice to have some sort of time line when to expect what.
Especially when some bugs will be fixed that have been reported.
This could be done by setting up some milestones.

**Comment by alex4998** [comment]
Thank you for the answers. I got the idea.
   Probably I have to start another thread to continue with the performance part of the question but as long as we touched base on that... I'm displaying a chart with just around 4000 points total and only 270 points visible at any given moment. It is limited by MinLimit/MaxLimit. Initial displaying is fast, no complaints but when I start scrolling it right and left it is quite sluggish. I didn't get deep into implementation details, but I would say with this amount of data everything should fly. I believe partially the sluggishness is caused by animation. By the way, how to turn it off? However partially it is because of how the visualization is implemented. If I increase the total number of points to around 100000, this is what I eventually need, it is hardly moving but why? I specify strict view limit min and max, technically it can ignore all the data besides those 270 points in the view plus probably a page forward and a page backward and I would say it should have the same performance if I have 1000 points or 1000000 points in this case.
   Thoughts? Suggestions?
